src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -1345,4 +1345,10 @@ static constexpr Property<Affinity> affinity{"AFFINITY"};
  */
 static constexpr Property<std::vector<std::string>, PropertyMutability::RO> execution_devices{"EXECUTION_DEVICES"};
 
+/**
+ * @brief Path to the file with model's weights.
+ *
+ * @note This property is used for weightless caching. Only used when ov::CacheMode Property is set to "OPTIMIZE_SIZE".
+ */
+static constexpr Property<std::string, PropertyMutability::RW> weights_path{"WEIGHTS_PATH"};
```

Comment:
I guess it should be added to python api files as well: [#1](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/pyopenvino/core/properties/properties.cpp) [#2](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/openvino/runtime/properties/__init__.py) [#3](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/tests/test_runtime/test_properties.py)

Reply:
Done.